Solutions/Ais.Net.Models/Ais/Net/Models/Abstractions/ITimestamp.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Ais.Net.Models.Abstractions;
+
+public interface ITimestamp
+{
+    public long? UnixTimestamp { get; }
```

Comment:
Unix timestamps come in two different units: seconds and milliseconds. I don't know which this is. Please could you rename it to make the units clear?

Reply:
I was simply using the field name that you had used to maintain continuity but can rename if desired?

https://github.com/ais-dotnet/Ais.Net/blob/3f0624713cd879f0234b167e9e77b899765e8843/Solutions/Ais.Net/Ais/Net/NmeaTagBlockParser.cs#L141